src/field/index.js

```diff
@@ -361,11 +375,21 @@ class Field {
                 this.setValue(name, fieldsValue[name], false);
             });
         } else {
+            // NOTE: this is a shallow merge
```

Comment:
add NOTE: 
we have two values  `a.b.c=1 ; a.b.d=2`,  and use setValues({a:{b:{c:3}}}) , then because of shallow merge  `a.b.d` will lost, we will get only `{a:{b:{c:3}}}`